File: Codigos/002-PrimeiroEstrutura/C/EstruturaSimples.c
#include <stdio.h>
#define alturaMaxima 225 //definido uma constate em C

typedef struct {
    int peso;       //peso em quilogramas
    int altura;     //altura em centímetros
} PesoAltura;

int main() {
    PesoAltura pessoa1;
    pessoa1.peso = 80;
    pessoa1.altura = 185;

    printf("Peso: %i, Altura: %i. ", pessoa1.peso, pessoa1.altura);

    if(pessoa1.peso > alturaMaxima) printf("Altura acima da máxima.\n");
    else printf("Altura abaixo da máxima.\n");
    
    return 0;
}

File: Codigos/002-PrimeiroEstrutura/Java/EstruturaSimples.java
class PesoAltura {
    int peso;       //peso em quilogramas
    int altura;     //altura em centímetros
}

class EstruturaSimples {
    public static final int alturaMaxima = 225;
    public static void main(String[] args) {
        PesoAltura pessoa1 = new PesoAltura();
        pessoa1.peso = 80;
        pessoa1.altura= 185;

        System.out.print("Peso: " + pessoa1.peso + ", Altura " + pessoa1.altura + ". ");
        
        if(pessoa1.altura > alturaMaxima) System.out.println("Altura acima da máxima.");
        else System.out.println("Altura abaixo da máxima.");
    }
}
